chunky/src/java/se/llbit/chunky/block/MinecraftBlockProvider.java

```diff
@@ -2428,12 +2428,7 @@ public Block getBlockByTag(String namespacedName, Tag tag) {
         // as of 20w13a (1.16), the jigsaw block supports 12 orientations saved in the orientation tag
         Tag orientation = tag.get("Properties").get("orientation");
         if (orientation.isError()) {
-          return new OrientedTexturedBlock(
-              "jigsaw",
-              BlockProvider.facing(tag, "up"),
-              Texture.jigsawSide,
-              Texture.jigsawTop,
-              Texture.jigsawBottom);
+          return new JigsawBlock("jigsaw", BlockProvider.facing(tag, "north"));
```

Comment:
The default value changed, was that intentional?

Reply:
Nope, misremembered the default orientation.